website/docs/docs/build/packages.md

```diff
@@ -96,12 +96,12 @@ By default, `dbt deps` will not include prerelease versions when resolving packa
 - Explicitly specifying a prerelease version in your `version` criteria
 - Setting `install-prerelease` to `true`, and providing a compatible version range
 
-Both of the following configurations would successfully install `0.4.5a2` of `dbt_artifacts`:
+Both of the following configurations would successfully install `0.4.5-a2` of `dbt_artifacts`:
```

Comment:
```suggestion
For example, both of the following configurations would successfully install `0.4.5-a2` for the [`dbt_artifacts` package](https://hub.getdbt.com/brooklyn-data/dbt_artifacts/latest/):
```

Reply:
hey @dbeatty10 , looks awesome and i'm suggesting the above for extra guidance. haven't merged your pr yet and would love to hear your quick thoughts if you're ok with this!